Adds Rakuten 24 Core Web Vitals case study (#8516)
GoogleChrome/web.dev: src/site/content/en/blog/rakuten/index.md

description: >
  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.
date: 2022-08-19
hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/uB1PUfGOXOhrqUBW3tfO.jpg
thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/OLBDJ1YEaVWGkR5lqNyR.jpg
alt: A banner image with the Rakuten 24 logo and branded artwork.
authors:
  - linhduong
  - shogokashiwase
  - hayounglee
  - ryunosukeakiba
tags:
  - blog
  - case-study
  - web-vitals
---

[Rakuten 24](https://24.rakuten.co.jp/) is an online store that collaborates with both major multinational and domestic consumer goods manufacturers to offer a great variety of daily necessities including healthcare, beverage, pet supplies, baby products, and more. This store is provided by Rakuten Group, Inc.—a global leader in internet services, and is among the top performers on their digital marketplace platform in Japan.

Understanding the impact of web performance on user experience, the Rakuten 24 team has been continuously measuring, optimizing and monitoring Core Web Vitals and other metrics.

> As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.

Author: ```suggestion
As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**. However, they are still working on Cumulative Layout Shift (CLS) improvements.
```

---
layout: post
title: How Rakuten 24's investment in Core Web Vitals increased revenue per visitor by 53.37% and conversion rate by 33.13%
subhead: >
  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.
description: >
  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.

> date: 2022-08-15

Author: ```suggestion
date: 2022-08-19
```

hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/uB1PUfGOXOhrqUBW3tfO.jpg
thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/OLBDJ1YEaVWGkR5lqNyR.jpg
alt: A banner image with the Rakuten 24 logo and branded artwork.
authors:
  - linhduong
  - shogokashiwase
  - hayounglee
  - ryunosukeakiba
tags:
  - blog
  - case-study
  - web-vitals
---

[Rakuten 24](https://24.rakuten.co.jp/) is an online store that collaborates with both major multinational and domestic consumer goods manufacturers to offer a great variety of daily necessities including healthcare, beverage, pet supplies, baby products, and more. This store is provided by Rakuten Group, Inc.—a global leader in internet services, and is among the top performers on their digital marketplace platform in Japan.

Understanding the impact of web performance on user experience, the Rakuten 24 team has been continuously measuring, optimizing and monitoring Core Web Vitals and other metrics.

> As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.

Reviewer: In general the styleguide suggests not having content in bold. These stats might be clearer using the stats component.

> As a result, **over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP)**.

Reviewer 2: @malchata Shu Hirasaka contacted me via email regarding your concern about the lack of mention about CLS in the article. This is because we are still working on improving CLS so he suggested adding a sentence about it.

How about adding it here like this?
"As a result, over 75% of their users are experiencing good Largest Contentful Paint (LCP), First Input Delay (FID) and First Contentful Paint (FCP). They are still working on Cumulative Layout Shift (CLS) improvements."

If it doesn't work for you, feel free to suggest a better place to add this sentence.

## Highlight the opportunity

Though optimizing web performance is a smart investment to improve user experience and business growth, Rakuten 24 team understands how difficult it can be to convince stakeholders to adopt Core Web Vitals and focus on web performance. They believe that showing stakeholders exactly what kind of Return on Investment (ROI) performance optimization can bring is the best way to get them on board.

Being a relatively new and independent service, Rakuten 24 used their advantage of flexibility to take up the challenge. They believe that the result of their case study would help them make more data-oriented decisions in the future, as well as help other developers measure the impact of their work and convince their stakeholders that improving performance is worth the investment.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ij6fET9W23vj1IOb3e6O.jpg", alt="Example screenshots of Rakuten 24's homepage, with mobile device frames around each screenshot.", width="800", height="537" %}
  <figcaption>
    Some example screenshots of Rakuten 24's homepage.
  </figcaption>
</figure>

> ## The approach they used

Reviewer: Can we do something about these stacked headings? They don't clearly intro the section when you read this in the preview. I actually think you could move everything up a level, lose "The approach they used".

Then make an h2 of "Web performance optimization techniques" or "Approach taken for web performance optimization", the follow that with a sentence explaining these were the techniques used. 

Same for the analysing section. THat would move everything up a level and prevent these confusing stacked headings.

Author: @rachelandrew, I pushed a change that flattens the structure of the document. Let me know if it works for you.

@duongkhlinh, we decided to reorganize some headings due to the feedback Rachel had above. Please advise on whether this will work for you.

Reviewer: @malchata that looks better to me.

Reviewer 2: @malchata Flattening the structure works for me. However, I have 2 suggestions:

**1. Can you consider adding a sentence to the end of the "Highlight the opportunity" section?**
Something like "Find out how they did it in this post" or "Find out the approach they used below".
**Reason:** It feels a bit sudden to jump straight from "Highlight the opportunity" to "Optimize JavaScript and resources". Adding the above sentence would make the flow more natural in my opinion.

**2. Can you consider changing the title of the analyzing section as below?**
- "Performance measuring, analyzing, and monitoring" → "Performance measuring"
- "Analyzing" → "Performance Analyzing"
- "Monitoring" → "Performance Monitoring"
 
**Reason:** "Performance measuring, analyzing, and monitoring" section now only talks about measuring so changing it to "Performance measuring" seems easier to understand. Adding "performance" to "analyzing" and "monitoring" makes it clearer too I think.

### Web performance optimization

#### Optimize JavaScript and resources

- Eliminate render-blocking resources.
- Split code and use [dynamic `import()`](https://v8.dev/features/dynamic-import).
- Split all content into separate parts and lazy load below-the-fold HTML files.
- Execute and load JavaScript on demand.
- Identify slow JavaScript resources and optimize the loading process by using the async attribute on `<script>` tags and establishing early connections to important origins (Resource hints such as `dns-prefetch`, `preconnect`, and `preload`).
- [Remove unused code](/remove-unused-code/), and [minify and compress code](/reduce-network-payloads-using-text-compression/).

> - Use a CDN.

Reviewer 2: @malchata I'm wondering if this should be `[Use a CDN](/content-delivery-networks/)` or did you leave the URL out on purpose? If you left it out on purpose, that's fine too. Just checking in case!

Author: Good catch. I updated this in a commit.

> - Use a CDN.

Author: ```suggestion
- [Use a CDN](/content-delivery-networks/).
```

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/o5Nd3IV6Yof3mfr3tVH3.jpg", alt="Rakuten 24's web vitals tracking integration flow. The first step is to integrate the web-vitals library by adding the script to the Rakuten 24 website. After that, web vitals can be measured from real user metrics, and the data is sent to Rakuten 24's in-house data collection tool.", width="800", height="495" %}
</figure>

#### Analyzing

The team analyzed the collected field data to determine if there is any correlation between the Core Web Vitals and the key business metrics. They found out that the converted users tend to experience a better LCP than the users who didn’t convert.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/xWYjqq9sEJ2yQlDxl6fY.jpg", alt="A comparison of users who converted versus those who didn't by LCP. The user group who converted more frequently experienced a lower LCP.", width="800", height="391" %}
</figure>

The collected data also revealed that:

- A good LCP can lead to an increase of up to **61.13% in conversion rate**, **26.09% in revenue per visitor**, and **11.26% in average order value**.

> - A good FID can lead to an increase of up to **55.88% in conversion rate** compared to the overall average data.

Reviewer 2: @malchata Can `compared to the overall average data` be moved to a new line? Initially I separated this to a new line because this is the common part of both line 91 and 92 (A good LCP can ... compared to the overall average data, A good FID can .. compared to the overall average data). If you think it's better this way, that's fine too.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/nd87NaZ2i9cG905ChUdU.jpg", alt="A screenshot of Rakuten 24's internal performance monitoring dashboard for each of the Core Web Vitals (LCP, CLS, and FID).", width="800", height="354" %}
  <figcaption>
    Performance monitoring dashboard.
  </figcaption>
</figure>

### A/B test

Believing that A/B tests are a good way to measure the business impact of performance optimizations, the team optimized one of their landing pages for Core Web Vitals then compared the optimized version with the original page via A/B test for a month. They picked a landing page with significant traffic and conversion so that the test could achieve meaningful results. During the test duration, 50% of the traffic was sent to the optimized landing page (version A), and 50% was sent to the original page (version B). The only difference between version A and version B was that version A was optimized for Core Web Vitals and there were no other functional or visual differences.

<figure>

>   {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pN5i17Rh7VxrObYlgdiu.jpg", alt="A screenshot of a mobile A/B test for the Rakuten 24 website. Each version was visually and functionally the same, with version A optimized for better Core Web Vitals.", width="800", height="1129" %}

Reviewer 2: @malchata Hi, this image looks too big on PC screen so can you replace it with the following?
https://drive.google.com/file/d/1Yoktvq9Oo2P9yeiMWMws_bBntOXST_-A/view?usp=sharing
(I originally meant to make this part into a 2-column content like text-image but if this cannot be two columns, I think an image with background on both sides will work better)

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/ngLxHt0V3Ubss5gkT05t.jpg", alt="A startup comparison of the Rakuten 24 homepage. Version A is optimized for better loading, loading within 1.6 seconds, compared to Version B, which loaded within 2 seconds.", width="800", height="488" %}
  <figcaption>
    Mobile load test result of version A and version B.
  </figcaption>
</figure>

Comparing the optimized version A to the unoptimized version B, Rakuten 24 found that version A brings about:

- 53.37% increase in revenue per visitor.
- 33.13% increase in conversion rate.
- 15.20% increase in average order value.
- 9.99% increase in average time spent.
- 35.12% reduction in exit rate.

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/m9YXxAbL0Cc3qqblK4Wu.jpg", alt="A screenshot of Core Web Vitals improvements for the Rakuten 24 homepage. The stats are a 53.37% increase in revenue per visitor, 33.13% increase in conversion rate, 15.2% increase in average order value, 9.99% increase in average time spent on pages, and 35.12% reduction in exit rate.", width="800", height="285" %}
</figure>

## Conclusion

> Web performance optimization is challenging but rewarding. By taking a data-driven approach, Rakuten 24 has successfully delivered better user experience as well as measured positive impact on their business. Understanding that this is just a part of the journey not the destination, they will continue improving their website to provide online shoppers with even more delighting experience.

Reviewer: ```suggestion
Web performance optimization is challenging but rewarding. By taking a data-driven approach, Rakuten 24 has successfully delivered better user experience as well as measured positive impact on their business. Understanding that this is just a part of the journey not the destination, they will continue improving their website to provide online shoppers with more delightful experiences.
```

> Optimization requires a joint effort and developers do not have to be alone on this journey. By sharing their struggles and achievements, Rakuten 24 hopes that more developers can start utilizing Core Web Vitals data to develop a mutual understanding with stakeholders then work together towards high-quality user experience and business growth.

Reviewer: ```suggestion
Optimization requires a joint effort and developers do not have to be alone on this journey. By sharing their struggles and achievements, Rakuten 24 hopes that more developers can use Core Web Vitals data to develop a mutual understanding with stakeholders then work together towards high-quality user experience and business growth.
```

---
layout: post
title: How Rakuten 24's investment in Core Web Vitals increased revenue per visitor by 53.37% and conversion rate by 33.13%
subhead: >
  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.
description: >
  By measuring real users' Web Vitals, Rakuten 24 also found that a good Largest Contentful Paint (LCP) can lead to a conversion rate increase of 61.13%.

> date: 2022-08-23

Author: ```suggestion
date: 2022-08-24
```
